Vagrantfile~

```diff
@@ -0,0 +1,29 @@
+# -*- mode: ruby -*-
```

Comment:
You should remove this swap file
Add an entry to the gitignore file to avoid this happening again